server/server.go

```diff
@@ -642,3 +660,45 @@ func checkBranchTranslationCmdF(args []string, w http.ResponseWriter, slashComma
 
 	return nil
 }
+
+func pipelineTriggerCmdF(args []string, w http.ResponseWriter, slashCommand *MMSlashCommand) error {
+	const colorErr = "#ee2116"
+	const colorSuccess = "#0060aa"
+
+	if len(args) < 1 {
+		msg := "You need to set at least one pipeline name to trigger."
```

Comment:
Propose adding a space before appending to `msg` below. (0/5 if this should instead be a newline.)
```suggestion
		msg := "You need to set at least one pipeline name to trigger. "
```

Reply:
Yes lets add space here.